cmd/tailscale/cli/network-lock.go

```diff
@@ -99,6 +100,18 @@ func runNetworkLockInit(ctx context.Context, args []string) error {
 		return err
 	}
 
+	// Common mistake: Not specifying the current node's key as one of the trusted keys.
```

Comment:
optional: I think this could be done more succinctly using `x/exp/slices` `slices.ContainsFunc` but I'm not sure if it's worth changing that now.